dbcon/joblist/lbidlist.cpp

```diff
@@ -759,12 +807,13 @@ bool LBIDList::CasualPartitionPredicate(const BRM::EMCasualPartition_t& cpRange,
                     uint64_t val = *(int64_t*)MsgDataPtr;
                     value = static_cast<int64_t>(val);
                 }
+
                 case 16:
                 {
                     unsigned __int128 val;
-                    bigValuePod = reinterpret_cast<dataconvert::Int128Pod_t*>(&val);
-                    bigValuePod->lo = *reinterpret_cast<const uint64_t*>(MsgDataPtr);
-                    bigValuePod->hi = *(reinterpret_cast<const uint64_t*>(MsgDataPtr) + 1);
+                    int128Ptr = reinterpret_cast<uint64_t*>(&val);
```

Comment:
This approach surely works. 
I see as a 16-bytes read from BS::buf() and subsequent int128 cast though. It's up to you to decide should we proceed with this version or not.

Reply:
I agree. A direct cast to int128_t* should work just fine. I am changing this.